doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Configuration server                                    | Configuration server (confd) is a software module that mediates access to the configuration database. Also, the server node on which this module runs.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Credit                                                  | A credit is a measure of a group of updates. A credit is a pair (nr, size), where nr is the number of updates and size is total size in bytes of modified regions.                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
This needs to be refined. Current description is not good.

Reply:
Updated.